### PR TITLE
Disabled Dropdown is showing Menu on hover

### PR DESCRIPTION
**Pull request by yociduo**
Fix: Disabled Dropdown is showing Menu on hover #7018

### CODE REVIEW

**Comment by afc163** [comment]
Please clean up the extra commits.

**Comment by afc163** [comment]
Please upgrade the API documentation too.

**Comment by yociduo** [comment]
updated

File: components/dropdown/dropdown.tsx [hunk 1]
```diff
@@ -44,18 +45,20 @@ export default class Dropdown extends React.Component<DropDownProps, any> {
...
     const fixedModeOverlay = cloneElement(overlay as any, {
       mode: 'vertical',
     });
     return (
       <RcDropdown
         transitionName={this.getTransitionName()}
-        {...this.props}
+        trigger={disabled ? [] : trigger}
         overlay={fixedModeOverlay}
+        {...this.props}
```
Comment (benjycui) [inline components/dropdown/dropdown.tsx]: 这样 `this.props.overlay` 将会覆盖 `fixedModeOverlay`

Comment (benjycui) [inline components/dropdown/dropdown.tsx]: I am handling this.

Comment (afc163) [inline components/dropdown/dropdown.tsx]: Bug, I should write a test case for it.

Comment (benjycui) [inline components/dropdown/dropdown.tsx]: e0bf1ce

**Pull request merged**